CommonAssemblyInfo.cs

```diff
@@ -15,8 +15,8 @@
 [assembly: AssemblyProduct("Virto Commerce 2.13")]
 [assembly: AssemblyCopyright("Copyright © VirtoCommerce 2011-2020")]
 
-[assembly: AssemblyFileVersion("2.13.63.0")]
-[assembly: AssemblyVersion("2.13.63.0")]
+[assembly: AssemblyFileVersion("2.13.64.0")]
+[assembly: AssemblyVersion("2.13.64.0")]
```

Comment:
Do not need to increase version in this commit, as it is to the dev branch.
It will be increased before merge to master - in Assembly info and in module.manifest